File: Mars-Images/MarsImageCaptionView.m
//
//  MarsImageCaptionView.m
//  Mars-Images
//
//  Created by Mark Powell on 1/8/14.
//  Copyright (c) 2014 Powellware. All rights reserved.
//

#import "MarsImageCaptionView.h"

@implementation MarsImageCaptionView

- (id)initWithPhoto:(id<MWPhoto>)photo {
    self = [super initWithPhoto:photo];
    return self;
}

- (CGSize)sizeThatFits:(CGSize)size {
    if (size.width > 100) {
        return [super sizeThatFits:size];
    }
    return CGSizeMake(0,0);
}

@end

File: MarsImagesIOS/MarsImageCaptionView.swift
//
//  MarsImageCaptionView.swift
//  MarsImagesIOS
//
//  Created by Mark Powell on 7/29/17.
//  Copyright © 2017 Mark Powell. All rights reserved.
//

import UIKit
import MWPhotoBrowser

class MarsImageCaptionView : MWCaptionView {
    
    //if it's still fine looking after testing, just let the caption always display even when in a compressed width
    override func sizeThatFits(_ size: CGSize) -> CGSize {
//        if size.width > 100 {
            return super.sizeThatFits(size)
//        }
//        return CGSize(width: 0, height: 0)
    }
}
